distribution/docs/src/main/resources/_contents/_federation/federation-intro-contents.adoc

```diff
@@ -0,0 +1,131 @@
+The source ID in each metacard identifies the Source from which the metacard originated.
+
+The Catalog normalizes the incoming query into an OGC Filter format.
+When the query is disseminated by the Catalog Framework to the sources, each source is responsible for denormalizing the OGC Filter formatted query into the format understood by the external store that the source is acting as a proxy.
+This normalization/denormalization is what allows any endpoint to interface with any type of source.
+For example, a query received by the OpenSearch Endpoint can be executed against an OpenSearch Source.
+
+.Federation Data Flow
+[ditaa, federation-data-flow, png, ${image-width}]
+....
+           /------\
+           |Client|
+           |cDEF  |
+           \------/
+              ^
+              |
++-------------|------------------------------------------------------------\
+|DDF          |                                                       cCCC |
+|             v                                                            |
+|         /--------\   /-----------------\   /-------------------\         |
+|         |Endpoint|<->|Catalog Framework|<->|Federation Strategy|         |
+|         | cDEF   |   | cDEF            |   | cDEF              |         |
+|         \--------/   \-----------------/   \-------------------/         |
+|                                                     ^ ^                  |
+|                                                     | |                  |
+|                                          /----------/ \---------\        |
+|                                          |                      |        |
+| +----------------------------------------|----------------------|------+ |
+| |Enterprise Query                        v                      v      | |
+| |+--------------------------------------------+ +---------------------+| |
+| ||Local Query                                 | |Federated Query      || |
+| ||/------------------\ /---------------------\| |/-------------------\|| |
+| ||| Catalog Provider | | Connected Source(s) || ||Federated Source(s)||| |
+| |||  cDEF            | |  cDEF               || || cDEF              ||| |
+| ||\------------------/ \---------------------/| |\-------------------/|| |
+| |+-------^----------------------^-------------+ +----^----------^-----+| |
+| +--------|----------------------|--------------------|----------|------+ |
+\----------|----------------------|--------------------|----------|--------/
+           |                      |                    |          \-------------------\
+           |                      |                    |                              |
+           v                      v                    v                              v
+/=-------------------\ /----------------------\ /---------=------------\     /----------------------\
+|  Oracle Database   | |       Source A       | |       Source B       | ... |       Source X       |
```

Comment:
changed here. Did a global search and the other references related to Java version